src/bin/tikv-server.rs

```diff
@@ -38,27 +38,27 @@ extern crate signal;
 extern crate tikv;
 extern crate toml;
 
-mod signal_handler;
 #[cfg(unix)]
 mod profiling;
+#[macro_use]
+mod tikv_setup;
```

Comment:
Why not remove the prefix?

Reply:
The prefix looks fine to me, why do you want to remove it?

Reply:
It looks like a standalone binary.